connect/src/main/java/com/stripe/android/connect/AccountOnboardingView.kt

```diff
@@ -33,16 +40,55 @@ class AccountOnboardingView private constructor(
         StripeConnectWebViewContainerImpl(
             embeddedComponent = StripeEmbeddedComponent.ACCOUNT_ONBOARDING,
             embeddedComponentManager = embeddedComponentManager,
+            props = props?.toJs()?.toJsonObject(),
             listener = listener,
-            listenerDelegate = AccountOnboardingListenerDelegate
+            listenerDelegate = AccountOnboardingListenerDelegate,
         )
     )
 
     init {
+        context.withStyledAttributes(attrs, R.styleable.StripeAccountOnboardingView, defStyleAttr, 0) {
+            val props = AccountOnboardingProps(
+                fullTermsOfServiceUrl = getString(
+                    R.styleable.StripeAccountOnboardingView_stripeFullTermsOfServiceUrl
+                ),
+                recipientTermsOfServiceUrl = getString(
+                    R.styleable.StripeAccountOnboardingView_stripeRecipientTermsOfServiceUrl
+                ),
+                privacyPolicyUrl = getString(
+                    R.styleable.StripeAccountOnboardingView_stripePrivacyPolicyUrl
+                ),
+            )
+            webViewContainerBehavior.updateProps(
+                props = props.toJs().toJsonObject(),
+                merge = true
+            )
+        }
+
         webViewContainerBehavior.initializeView(this)
     }
+
+    /**
+     * Updates the component props to [props]. If [merge] is true, existing values will not be overwritten by
+     * null values.
+     */
+    fun updateProps(props: AccountOnboardingProps, merge: Boolean) {
```

Comment:
👀 `merge` is useful to specify whether props should be completely replaced by `props` or merged with existing values, e.g. with those declared in XML.

Reply:
removed as discussed